packages/main/test/pages/Menu.html

```diff
@@ -111,11 +111,14 @@ <h3 class="header-title">Event logger</h3>
 			if (item && item.text === "Open") {
 				menu.removeEventListener("ui5-before-open", fetch);
 				setTimeout(function() {
+					// item.removeAttribute("busy");
+					// item.removeAttribute("busy-delay");
```

Comment:
Should dis stay as comment?

Reply:
Not sure. I've left those for completeness, but those would intervene with the test execution.

Reply:
Done